kpi/tests/api/v2/test_api_asset_export_settings.py

```diff
@@ -0,0 +1,216 @@
+                'export_settings': updated_export_settings,
+            },
+            format='json',
+        )
+        assert response.status_code == status.HTTP_200_OK
+        assert self.asset_export_settings.count() == 1
+
+        data = response.json()
+        assert data['name'] == new_name
+        assert data['export_settings'] == updated_export_settings
+
+    def test_api_delete_asset_export_settings_for_owner(self):
+        response = self.client.post(
+            self.export_settings_list_url,
+            data={
+                'name': self.name,
+                'export_settings': self.valid_export_settings,
+            },
+            format='json'
+        )
+        assert response.status_code == status.HTTP_201_CREATED
+        data = response.json()
+        export_settings_url = data['url']
+
+        assert self.asset_export_settings.count() == 1
+
+        delete_response = self.client.delete(export_settings_url)
+        assert delete_response.status_code == status.HTTP_200_OK
+        assert self.asset_export_settings.count() == 0
+
+    def test_api_list_asset_export_settings_for_owner(self):
+        names = ['foo', 'bar', '🤡']
+        all_export_settings = [
+            self._create_foo_export_settings(name) for name in names
+        ]
+        response = self.client.get(self.export_settings_list_url)
+        assert response.status_code == status.HTTP_200_OK
+
+        data = response.json()
+        assert data['count'] == len(names) == self.asset_export_settings.count()
+
+        results = data['results']
+        for i, export_settings in enumerate(reversed(all_export_settings)):
+            assert results[i]['name'] == list(reversed(names))[i]
+            assert results[i]['uid'] == export_settings.uid
+
+    def test_api_list_asset_export_settings_without_perms(self):
+        self._create_foo_export_settings()
+
+        self._log_in_as_another_user()
+        response = self.client.get(self.export_settings_list_url)
+        assert response.status_code == status.HTTP_404_NOT_FOUND
+
+    def test_api_list_asset_export_settings_with_perms(self):
+        self._create_foo_export_settings()
+
+        self._log_in_as_another_user()
+        response = self.client.get(self.export_settings_list_url)
+        assert response.status_code == status.HTTP_404_NOT_FOUND
+
+        self.asset.assign_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS)
+        response = self.client.get(self.export_settings_list_url)
+        assert response.status_code == status.HTTP_200_OK
+
+        data = response.json()
+        assert data['count'] == self.asset_export_settings.count() == 1
+
+    def test_api_detail_asset_export_settings_without_perms(self):
+        export_settings = self._create_foo_export_settings()
+        url = self.__get_detail_url(export_settings.uid)
+
+        self._log_in_as_another_user()
+        response = self.client.get(url)
+        assert response.status_code == status.HTTP_404_NOT_FOUND
```

Comment:
would you mind adding a test to make sure `manage_asset` does what it's supposed to do?